provider/oauth2/tests.py

```diff
@@ -113,7 +114,7 @@ def test_implicit_flow(self, requested_scope, expected_scope):
         response = self.client.post(self.auth_url2(), {'authorize': True})
         fragment = urlparse.urlparse(response['Location']).fragment
         auth_response_data = {k: v[0] for k, v in urlparse.parse_qs(fragment).items()}
-        self.assertEqual(auth_response_data['scope'], expected_scope)
+        self.assertItemsEqual(auth_response_data['scope'].split(' '), expected_scope.split(' '))
```

Comment:
This was necessary due to tox setting PYTHONHASHSEED, rendering the ordering of these items no longer idempotent. Since that's a valid thing to do I made the tests support it.